uxarray/remap/dataarray_accessor.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Union
+
+from uxarray.remap.nearest_neighbor import _nearest_neighbor_uxda
+from uxarray.remap.inverse_distance_weighted import (
+    _inverse_distance_weighted_remap_uxda,
+)
+
+if TYPE_CHECKING:
+    from uxarray.core.dataset import UxDataset
+    from uxarray.core.dataarray import UxDataArray
+
+from uxarray.grid import Grid
+
+
+class UXDataArrayRemapAccessor:
```

Comment:
Can we add a `__repr__` like our `subset` accessor?

https://github.com/UXARRAY/uxarray/blob/b6bb18b730eb64ebf5f47e7316247feea27b8ae7/uxarray/subset/dataarray_accessor.py#L18-L28

Reply:
Right I forgot about this, sure thing!